pkg/controller/provider/controller.go

```diff
@@ -52,7 +53,9 @@ import (
 
 const (
 	// Name.
-	Name = "provider"
+	Name               = "provider"
+	OvaTimeout         = 10
```

Comment:
it would be better to make this a duration as the const below in order to have units